build.assets/genproto.sh

```diff
@@ -13,20 +13,19 @@ main() {
   trap 'rm -fr github.com' EXIT   # don't leave github.com/ behind
   rm -fr api/gen/proto gen/proto  # cleanup gen/proto folders
 
-  # Generate Gogo protos (default).
-  buf generate --template=buf-gogo.gen.yaml
+  # Generate Gogo protos.
+  buf generate --template=buf-gogo.gen.yaml api/proto
+  buf generate --template=buf-gogo.gen.yaml proto
 
   # Generate protoc-gen-go protos (preferred).
   # Add your protos to the list if you can.
   buf generate --template=buf-go.gen.yaml \
     --path=api/proto/teleport/devicetrust/ \
     --path=api/proto/teleport/loginrule/ \
     --path=proto/teleport/lib/multiplexer/
+  buf generate --template=lib/prehog/buf.gen.yaml lib/prehog/proto
```

Comment:
Possibly?

Because this is Go it still follows go_package, so we should be able to control the output path through that and have a single gen file.

```suggestion

  # Generate connect-go protos.
  buf generate --template=buf-connectgo.gen.yaml lib/prehog/proto
```

Reply:
I'll move `lib/prehog/buf.gen.yaml` in the PR that gets rid of `make grpc-teleterm`.